### PR TITLE
Add support for minor versions of nop commerce

### PR DESCRIPTION
**Pull request by denisburlacu**

### CODE REVIEW

File: src/Libraries/Nop.Core/NopVersion.cs [hunk 1]
```diff
@@ -9,5 +9,9 @@ public static class NopVersion
         /// Gets or sets the store version
         /// </summary>
         public static string CurrentVersion => "4.40";
+
+        public static string MinorVersion => "0";
+
+        public static string FullVersion => CurrentVersion + "." + MinorVersion;
```
Comment (skoshelev) [inline src/Libraries/Nop.Core/NopVersion.cs:15]: Hi @denisburlacu Better to use string format: $"{CurrentVersion}.{MinorVersion}"

Comment (denisburlacu) [reply src/Libraries/Nop.Core/NopVersion.cs:15]: ok will adjust, actually make sense to do for all fiedls here something like
public static string FullVersion {get} = $"{CurrentVersion}.{MinorVersion}", and based on this interpolation string will be executed only once

Comment (skoshelev) [reply src/Libraries/Nop.Core/NopVersion.cs:15]: I agree, you may make changes and I'll merge your code.

Comment (denisburlacu) [reply src/Libraries/Nop.Core/NopVersion.cs]: @skoshelev i pushed a new commit can you please have a look

**Pull request merged**